### PR TITLE
upd(obs-backgroundremoval): added repology

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/obs-backgroundremoval/obs-backgroundremoval.pacscript [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
   sed -i "s;LIBDIR}/obs-plugin;LIBDIR}/x86_64-linux-gnu/obs-plugin;" CMakeLists.txt
   # fix plugin crashes OBS
   sed -i "s/std::string(newModel);/newModel;/" src/background-filter.cpp
 }
 
 build() {
-  mkdir build && cd build
-  cmake -j$(nproc) -DobsIncludePath=/usr/include/obs -DCMAKE_INSTALL_PREFIX:PATH=$STOWDIR/$name/usr .. > /dev/null && cmake --build . -j$(nproc) > /dev/null
+  mkdir build && cd build || exit
+  cmake -j"$(nproc)" -DobsIncludePath=/usr/include/obs -DCMAKE_INSTALL_PREFIX:PATH="${STOWDIR}"/"${name}"/usr .. > /dev/null && cmake --build . -j"$(nproc)" > /dev/null
```
Comment (Henryws) [inline packages/obs-backgroundremoval/obs-backgroundremoval.pacscript:32]: cmake doesn't recognize the `-j` flag

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/obs-backgroundremoval/obs-backgroundremoval.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/obs-backgroundremoval/obs-backgroundremoval.pacscript:32]: Thank you

**Comment by Zahrun** [comment]
Since https://github.com/royshil/obs-backgroundremoval/commit/cc9d4a5711f9388ed110230f9f793bb071577a23, the build fails. I commented to the author. Let’s wait for that to be fixed.

**Comment by Zahrun** [comment]
Btw, the CI build fails but says success

**Comment by cat-master21** [comment]
Curious @Zahrun why is language-pack-en[-base] installed as it does almost nothing see [filelist](https://packages.ubuntu.com/jammy/all/language-pack-en-base/filelist) while Debian installs the needed mo file as it happens.

**Comment by Zahrun** [comment]
Sorry all, I got confused between the `-git` version and this version.
This version compiles fine.

@cat-master21, indeed `language-pack-en` was not required and I could confirm the plugin works on debian too.

**Comment by Henryws** [comment]
CI fails, look at the logs.

**Comment by Zahrun** [comment]
> CI fails, look at the logs.

I committed again the same files, and it looks fine now

File: packages/obs-backgroundremoval/obs-backgroundremoval.pacscript [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
 prepare() {
   # install required dependency for codename > focal
-  codename=$(lsb_release -cs)
-  case $codename in
+  codename="$(lsb_release -cs)"
+  case "${codename}" in
     focal) ;;
 
     *)
       sudo apt install libsimde-dev
```
Comment (Henryws) [inline packages/obs-backgroundremoval/obs-backgroundremoval.pacscript:19]: This should be appended to `depends`

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/obs-backgroundremoval/obs-backgroundremoval.pacscript (hunk 2 above)
Comment (Zahrun) [reply packages/obs-backgroundremoval/obs-backgroundremoval.pacscript]: True, thank you

**Pull request merged**